### ISSUE TITLE
Load GPS-Model only once

### ISSUE THREAD

**Issue by wlbragg** [c172p-team/c172p#1255]
<s>It is a fact that loading the GPS-Model twice for lighting effect simplicity is costing us load time and overhead. So we need to load it only once and dynamically reposition it to the alternate "Float" variant location when called for.</s>
I had it that way originally and had trouble getting the interior shadow positioned correctly but understand it a bit better and should be able to get it close enough.

**Comment by onox** [comment]
@wlbragg This issue is a duplicate of some other issue?

**Comment by wlbragg** [comment]
If it is I couldn't find it. I thought i might have already opened an issue for this but couldn't find it.

**Comment by wlbragg** [comment]
@legoboyvdlp  @gilbertohasnofb 

Hum, I just tried to verify the increase in load time by having these two model loading tag pairs and it had literally no time difference if I commented out the amphibious model loading of the GPS. I would appreciate it if someone would help verify this. Just comment out the block that loads the model at a different offset for the amphibious version and see if you see any difference in load time. It's located near line 158 in Models/c172p.xml

**Comment by gilbertohasnofb** [comment]
@wlbragg I think these things do not tend to be linear. If your CPU or GPU or memory is not struggling, you won't see a big difference, but if you are near the limit of your machine then this can really cap your performance. Also, keep in mind that we are always adding stuff to this project and the aircraft is becoming heavier and larger as time goes by, so we should avoid duplicates at all costs. My 2 cents.

**Comment by wlbragg** [comment]
I agree, but these are loaded conditionally and I am seeing no difference.
I went back in history to see how it was loaded and translated before but couldn't find it. Do you know how to load it once and then conditionally translate it? I'll figure it out eventually, but if you know how it would save me some time.

**Comment by gilbertohasnofb** [comment]
I don't remember how it was done before but wouldn't a translate animation solve this? The object is loaded once and the translation would get as condition whether a bushkit is being used?

**Comment by wlbragg** [comment]
Yeah, that is what I thought. A translate needs a property to apply to the factor, I think. So I would need to make a filter to drive a new property to a value of 1 if condition is bushkit = 4 (amphibious).
Then the translate would use that property to shift the y axis .1. I already determined that is the approximate shift needed.

**Comment by gilbertohasnofb** [comment]
do you need a filter though? Don't translate animations accept conditions? If not, then go for it

**Comment by wlbragg** [comment]
>Don't translate animations accept conditions?

I tried it briefly and it didn't appear to work. I'll take more time with it and make sure.

**Comment by gilbertohasnofb** [comment]
Maybe it doesn't. The wiki says that nearly all animations accept conditionals, but doesn't state which don't accept them. Nor do the conditional docs in FGData. So if you are sure you did not make a typo and the code still doesn't work then don't spend too much time with it and move on to the filter solution.
